packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -1013,3 +1013,9 @@ export const VIDEO_RESOLUTIONS = {
     }
   }
 };
+
+export const MAX_FRAMESIZES = {
+  [QUALITY_LEVELS.LOW]: 1620,
+  [QUALITY_LEVELS.MEDIUM]: 3600,
+  [QUALITY_LEVELS.HIGH]: 8192
+};
```

Comment:
Can we get comments on what these are in reference to quality? Not impacting, but a suggestion.

Reply:
Will add a link to https://en.wikipedia.org/wiki/Advanced_Video_Coding